gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/ResourceManager.java

```diff
@@ -263,4 +292,90 @@ public static ProjectListOption fields(ProjectField... fields) {
    * @throws ResourceManagerException upon failure
    */
   void undelete(String projectId);
+
+  /**
+   * Returns the IAM access control policy for the specified project. Returns null if the resource
+   * does not exist or if you do not have adequate permission to view the project.
+   *
+   * @see <a
```

Comment:
wrong [order](http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#tag) of tags.